### ISSUE TITLE
🐞 [Bug]: DAO proposal dialogue should close after voting

### ISSUE THREAD

**Issue by khaledyoussef24** [threefoldtech/tfgrid-sdk-ts#3142]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### which package/s did you face the problem with?

Dashboard

### What happened?

DAO proposal dialogue should close after voting

### Steps To Reproduce

1- make a proposal
2- go to voting
3- vote with yes
4- choosing farm dialogue will open
5- dialogue is not closed even after voting

### which network/s did you face the problem on?

Dev, QA, Test, Main

### version

2.5.2

### Relevant screenshots/screen records

![telegram-cloud-photo-size-4-5830174116274094942-y](https://github.com/user-attachments/assets/a18f6e50-79b4-48b2-930c-085bafb6abe9)

### Relevant log output

```console
na
```

**Comment by zaelgohary** [comment]
Work completed:

Changed openVDialog value once voting is done.

**Comment by A-Harby** [comment]
Verified, Devnet c609551.

The dialogue is closed after the voting is done.

[screen-capture.webm](https://github.com/user-attachments/assets/9cbfb1a3-294c-46a4-aa8a-b7bba757ad82)

But I have opened another issue to reflect the changes on the page: https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3239.
